subprojects/docs/src/docs/userguide/what_is_gradle.adoc

```diff
@@ -12,133 +12,114 @@
+Tasks contain the logic that implements builds.
+In most use cases, you'll use or extend existing tasks, like the built-in Java `test` task.
+Gradle provides tasks that implement many common build system goals, like running tests and creating archives.
+Plugins provide even more types of tasks.
 
 Tasks themselves consist of:
 
- * Actions — pieces of work that do something, like copy files or compile source
- * Inputs — values, files and directories that the actions use or operate on
- * Outputs — files and directories that the actions modify or generate
+ * Actions: pieces of work that do something, like copy files or compile source
+ * Inputs: values, files and directories that the actions use or operate on
+ * Outputs: files and directories that the actions modify or generate
 
-In fact, all of the above are optional depending on what the task needs to do. Some tasks — such as the <<base_plugin#sec:base_tasks,standard lifecycle tasks>> — don't even have any actions. They simply aggregate multiple tasks together as a convenience.
+=== Plugins
 
-NOTE: You choose which task to run. Save time by specifying the task that does what you need, but no more than that. If you just want to run the unit tests, choose the task that does that — typically `test`. If you want to package an application, most builds have an `assemble` task for that.
+Plugins provide a means of reusing logic and configuration across multiple projects.
+With plugins, you can write a task once and use it in multiple builds. Or you can store common configuration,
+like logging, dependencies, and version management, in one place. This reduces duplication in
+build scripts.
 
-One last thing: Gradle's <<more_about_tasks#sec:up_to_date_checks,incremental build>> support is robust and reliable, so keep your builds running fast by avoiding the `clean` task unless you actually do want to perform a clean.
+=== Build Phases
 
-=== 3. Gradle has several fixed build phases
+Gradle evaluates and executes build scripts in three *build phases* of the <<build_lifecycle#build_lifecycle,Build Lifecycle>>:
 
-It's important to understand that Gradle evaluates and executes build scripts in three phases:
-
- 1. Initialization
-+
+Initialization::
 Sets up the environment for the build and determine which projects will take part in it.
 
- 2. Configuration
-+
-Constructs and configures the task graph for the build and then determines which tasks need to run and in which order, based on the task the user wants to run.
+Configuration::
+Constructs and configures the task graph for the build. Determines which tasks need to run and in which order, based on the task the user wants to run.
 
- 3. Execution
-+
+Execution::
 Runs the tasks selected at the end of the configuration phase.
 
-These phases form Gradle's <<build_lifecycle#build_lifecycle,Build Lifecycle>>.
-
-[NOTE]
-.Comparison to Apache Maven terminology
```

Comment:
Are these covered in the migrating from Maven chapter already?

Reply:
I believe so. Should I add a link to that chapter here, since so many potential users have Maven experience?

Reply:
Yeah, that might be a good idea here